chore(docs): What is CDKTF and Interoperability (#1065)
hashicorp/terraform-cdk: website/docs/cdktf/concepts/interoperability-workflows.html.md

---
layout: "docs"
page_title: "HCL Interoperability"
sidebar_current: "cdktf"

> description: "Learn how you can use configurations written in HCL and configurations written in CDK for Terraform together."

Reviewer: ```suggestion
description: "Use configurations written in HCL and configurations written in CDK for Terraform together to define and provision infrastructure."
```

slightly stronger for SEO to lead with the verb and avoid "learn to, etc."

Author: oh I didn't know that! I'll keep that in mind for the future - thank you!
